### DISCUSSION TITLE
Is there a Filterable Dropdown similar to Filterable Multiselect?

### DISCUSSION THREAD

**Discussion by jbrisbin-ipipeline** [carbon-design-system/carbon#14759] [Help] [answered]
Our users are requesting that single select Dropdowns/Selects have the same text-based filtering option that is available in Multiselect.  It seems like a fairly basic optimization of Dropdown, since FilterableMultiselect already incorporates it.  Has that been requested?  Does it already exist?

**Comment by tw15egan** [comment, marked as answer]
Yes, you can use the `Combobox` component with the `shouldFilterItem` prop passed in

https://react.carbondesignsystem.com/?path=/docs/components-combobox--overview#shouldfilteritem

**Reply by jbrisbin-ipipeline** [reply]
Is there a way to clear combobox text programmatically (in an onchange function from another component)?
